src/styles/index.scss

```diff
@@ -48,3 +48,60 @@
 //---------------------------------------
 @import '../components/ColorBlock/color-block';
 @import '../components/ColorGrid/color-grid';
+
+//---------------------------------------
+// Table component
+//---------------------------------------
+@import "narrative";
```

Comment:
We might want to do something different here, do you remember what/who added this file?

Reply:
@vpicone this was in particular regard to the way this table is being done on the narrative page on the Events site.  https://www.ibm.com/design/event/narrative

I was going to go back and at least change the name to be more reflective. My guess is that this was a bit of a hack from the 29 team. I wasn't sure how to implement it but saw that the Events say was calling some CSS for the table to work so I replicated what they did and wanted to ask you about it.

Mike also pointed out that in mobile that we need a line break between header and copy. Is there a better way we can achieve the same result? This is a pattern that I'm sure we'll want to duplicate across sites.

![IMG_4408 (1)](https://user-images.githubusercontent.com/33162280/76326178-a92f9e80-62be-11ea-8bf0-d42866d9d0f3.PNG)